lib/Slice.ark

```diff
@@ -0,0 +1,15 @@
+{
+    #made by Natendrtfm (on Github) with <3
+    (let sliceStr (fun (inputed startingIndex length) {
+        (assert (> (len inputed) length) "slice length must be inferior than argument length")
+        (assert (> (len inputed) 1) "argument length must be superior to 1")
+        (assert (> length 1) "slice length must be superior to 1")
+        (mut returnedString "")
+        (mut i startingIndex)
+        (while (< i length){
+            (set returnedString (+ returnedString (@ inputed i)))
```

Comment:
won't work without an assert since here i can have `startingIndex` out of bound of the string, and it will raise a runtime error